PSKoans/Koans/dbatools/AboutBackupDatabases.ps1

```diff
@@ -0,0 +1,167 @@
+#Requires -Modules dbatools
+using module PSKoans
+[Koan(Position = 1004)]
```

Comment:
Can you make sure all your koan topics include a `Module` property for the `koan` attribute? Like so:

```powershell
[Koan(Position = 1, Module = 'dbatools')]
```

You can also number from whatever you want. As long as you're referencing a module name, the numberings won't conflict :)

Reply:
Also, I'm gonna need you to put all these one level deeper; in the Koans/Modules/ folder under a dbatools folder. 🙂

Reply:
Done and pushed there. 👍